### DISCUSSION TITLE
Can't add a new sensor

### DISCUSSION THREAD

**Discussion by inv-samuel** [carla-simulator/carla#5789] [Q&A]
Hi
when i was trying to add a safe distance sensor in Carla (as per the doc : **https://carla.readthedocs.io/en/0.9.13/tuto_D_create_sensor/**) I'm facing some issue

**WARNING: Version mismatch detected: You are trying to connect to a simulator that might be incompatible with this API
WARNING: Client API version = 0.9.13
WARNING: Simulator API version = 0.9.13-1-g8854804f4-dirty
INFO: Found the required file in cache! Carla/Maps/TM/Town10HD_Opt.bin
INFO: Found the required file in cache! Carla/Maps/Nav/Town10HD_Opt.bin
basic agent set destination
ActorBlueprint(id=sensor.other.safe_distance,tags=[sensor, other, safe_distance])
Assertion failed: px != 0, file C:\workspace\carla\Build\boost-1.72.0-install\include\boost/smart_ptr/shared_ptr.hpp, line 734**

and the server crashed

Could anyone please help us to figure out what we need to do? Thanks for your work and best regards.

**Comment by devnandans** [comment]
I'm facing the same issue.
